src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
+   abstract class Machine {
+       public abstract void doWork();
+       public abstract void stopWork();
+   }
 public class MainApp {
-
+       public static void main(String[]args) {
```

Comment:
There is an unnecessary indentation before the `abstract` keyword at the beginning of `Machine` class declaration. Java code conventions suggest that class declarations should not be indented if they are not within another class or block.